sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAnalysisClient.cs

```diff
@@ -40,132 +40,7 @@ protected ConversationAnalysisClient()
-        ///       language: string, # Optional. The override language of a conversation item in BCP 47 language representation.
-        ///       modality: &quot;transcript&quot; | &quot;text&quot;, # Optional. Enumeration of supported conversational modalities.
-        ///       role: &quot;agent&quot; | &quot;customer&quot; | &quot;generic&quot;, # Optional. The role of the participant.
-        ///     }, # Required. The abstract base for a user input formatted conversation (e.g., Text, Transcript).
-        ///   }, # Required. The input ConversationItem and its optional parameters
-        ///   parameters: {
-        ///     projectName: string, # Required. The name of the project to use.
-        ///     deploymentName: string, # Required. The name of the deployment to use.
-        ///     verbose: boolean, # Optional. If true, the service will return more detailed information in the response.
-        ///     isLoggingEnabled: boolean, # Optional. If true, the service will keep the query for further review.
-        ///     stringIndexType: &quot;Utf16CodeUnit&quot;, # Optional. Specifies the method used to interpret string offsets. Set this to &quot;Utf16CodeUnit&quot; for .NET strings, which are encoded as UTF-16.
-        ///     directTarget: string, # Optional. The name of a target project to forward the request to.
-        ///     targetProjectParameters: Dictionary&lt;string, AnalysisParameters&gt;, # Optional. A dictionary representing the parameters for each target project.
-        ///   }, # Required. Input parameters necessary for a Conversation task.
-        ///   kind: Conversation, # Required. Enumeration of supported Conversation tasks.
-        /// }
-        /// </code>
-        /// </details>
-        /// 
-        /// Response Body:
-        /// 
-        /// <details><summary>ConversationalTaskResult</summary>Schema for <c>ConversationalTaskResult</c>:
-        /// <code>{
-        ///   result: {
-        ///     query: string, # Required. The conversation utterance given by the caller.
-        ///     detectedLanguage: string, # Optional. The system detected language for the query in BCP 47 language representation..
-        ///     prediction: {
-        ///       projectKind: &quot;Conversation&quot; | &quot;Orchestration&quot; | &quot;CustomConversationSummarization&quot;, # Required. The type of the project.
-        ///       topIntent: string, # Optional. The intent with the highest score.
-        ///     }, # Required. The prediction result of a conversation project.
-        ///   }, # Required. Represents a conversation analysis response.
-        ///   kind: ConversationResult, # Required. Enumeration of supported conversational task results
-        /// }
-        /// </code>
-        /// </details>
-        /// 
-        /// </remarks>
+        /// <include file="Docs/ConversationAnalysisClient.xml" path="doc/members/member[@name='AnalyzeConversationAsync(RequestContent,RequestContext)']/*" />
```

Comment:
What processes this? This is not authored correctly [according to C# XML doc comments](https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags#include) and, even if it were, I see nothing in this PR (or https://github.com/Azure/autorest.csharp/pull/2774) that would cause the doc file to be merged with the generate one and/or colocated in the nupkg. docs.microsoft.com (or learn.microsoft.com) is not the only target for these, so if this is some Docs-specific syntax, it will break IntelliSense documentation like when you hover over members in VS or VSCode.

Reply:
Never mind. I tried this out locally, and also realized the [docs](https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags#include) are misleading. What looks like attribute documentation is actually just describing the variables used therein. Seems the compiler does automatically merge this content into the final XML doc as well.